services/ui-src/src/components/reports/OverlayReportPage.tsx

```diff
@@ -134,6 +134,7 @@ export const OverlayReportPage = ({
                   entityType={entityType as EntityType}
                   key={entity.id}
                   locked={undefined}
+                  requiresMoreData={!standardEntities.length}
```

Comment:
suggestion: I think I'd prefer to call this parameter `disabled`. That's the pattern we've used elsewhere too when using data or user type, for example, to determine functionality. If only because then when you look at this parent component you know exactly what the parameter being passed to the sub-component does.
[ReportDrawer](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/blob/main/services/ui-src/src/components/drawers/ReportDrawer.tsx#L68)
[ModalOverlayReportPage](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/blob/main/services/ui-src/src/components/reports/ModalOverlayReportPage.tsx#L195)

Reply:
Chatted with Garrett about this - fell on the name hasStandards thats going to be passed. I didn't want to use locked here because locked can be used in other scenarios, and targets other buttons while I need something that happens specifically when theres no standards.